packages/blazor-workspace/Tests/NimbleBlazor.Tests.Acceptance.Shared/AcceptanceTestsBase.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.Playwright;
+using Xunit;
+
+namespace NimbleBlazor.Tests.Acceptance.Shared;
+
+[Collection(nameof(PlaywrightFixture))]
+public abstract class AcceptanceTestsBase
+{
+    private readonly PlaywrightFixture _playwrightFixture;
+
+    protected abstract Uri ServerAddress { get; }
+    protected abstract string ComponentLibraryInitializationTestJavaScript { get; }
```

Comment:
These two abstract read-only properties are the only real difference from before the refactor.